js/components/reorderList.js

```
@@ -103,14 +103,39 @@ Fliplet.FormBuilder.field('reorderList', {
+    /**
+     * Creates a deep clone of a value
+     * @param {*} value - value to clone
+     * @returns {*} cloned value
+     */
+    cloneDeep: function(value) {
+      if (value === null || typeof value !== 'object') return value;
+      if (value instanceof Date) return new Date(value.getTime());
+      if (Array.isArray(value)) return value.map(this.cloneDeep.bind(this));
+
+      if (typeof value === 'object') {
+        var cloned = {};
```

Biome (1.9.4)

[error] Use let or const instead of var.

A variable declared with var is accessible in the whole body of the function. Thus, the variable can be accessed before its initialization and outside the block where it is declared.
See MDN web docs for more details.
Unsafe fix: Use 'const' instead.


(lint/style/noVar)